.eslintrc

```diff
@@ -71,6 +72,7 @@
         "additionalEditorFeatures.js",
         "additionalPlayerFeatures.js",
         "adminTabFeatures.js",
+        "blogDashboard.js",
```

Comment:
@U8NWXD is it expected to add new files in this list?

Reply:
No. New files should follow the linter, not be excluded

Reply:
Removed.